sci-physics/fastjet/fastjet-3.4.0.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+FORTRAN_NEEDED=plugins
+PYTHON_COMPAT=( python3_{8..10} )
```

Comment:
I added `python3_11` to the compat list because it seems to compile fine with it.